themes/bootstrap3/less/components/sidebar.less

```diff
@@ -40,7 +40,6 @@
   .badge {
     flex-shrink: 0;
     max-height: 19px;
-    line-height: 1.1rem;
```

Comment:
I think this line was here to improve the centering of numbers in the badges. Using Firefox, here's what they look like in master:

![image](https://user-images.githubusercontent.com/309069/48219980-0a0e3600-e35c-11e8-859c-252ddf84e0fe.png)

...and here's what they look like in this branch:

![image](https://user-images.githubusercontent.com/309069/48219940-f1058500-e35b-11e8-8d8c-46e3bfd853f8.png)

Honestly, both of these look wrong to me, and all things being equal, having less styling is better than having more styling... but I wonder if we can do better. @crhallberg, any thoughts?

Reply:
Still seeing this issue; just flagging @crhallberg again in case he missed it the first time. Not sure if it matters, but it's a noticeable difference.